src/pyOpenMS/pxds/MSSpectrum.pxd

```diff
@@ -49,32 +88,32 @@ cdef extern from "<OpenMS/KERNEL/MSSpectrum.h>" namespace "OpenMS":
         void clear(bool clear_meta_data) nogil except + #wrap-doc:Clears all data (and meta data if clear_meta_data is true)
         void push_back(Peak1D)  nogil except + #wrap-doc:Append a peak
 
-        bool isSorted() nogil except +
+        bool isSorted() nogil except + #wrap-doc:Returns true if the spectrum is sorte by m/z
 
-        int findNearest(double) nogil except+
-        int findNearest(double, double) nogil except+
-        int findNearest(double, double, double) nogil except+
-        int findHighestInWindow(double, double, double) nogil except+
+        int findNearest(double) nogil except + #wrap-doc:Returns the index of the closest peak in m/z
+        int findNearest(double, double) nogil except + #wrap-doc:Returns the index of the closest peak in the provided +/- m/z tolerance window (-1 if none match)
```

Comment:
adding an exact description of parameters may help readers
i.e) params=(double mz, double tolerance)

Reply:
good point!